demo/server.js

```diff
@@ -1,67 +1,67 @@
-const dev = process.env.NODE_ENV !== 'production'
-const moduleAlias = require('module-alias')
+const dev = process.env.NODE_ENV !== 'production';
+const moduleAlias = require('module-alias');
 
 if (!dev) {
-  moduleAlias.addAlias('react', 'preact-compat')
-  moduleAlias.addAlias('react-dom', 'preact-compat')
+  moduleAlias.addAlias('react', 'preact-compat');
```

Comment:
Since you [removed `preact-compat`](https://github.com/FormidableLabs/react-live/pull/113/files#diff-7350bfb3587433477c29566176c70a1dL20) from the dependencies in this demo, do we need to remove this alias as well?

Reply:
Yes definitely!